src/main/resources/i18n/messages.properties

```diff
@@ -522,6 +522,8 @@ SharedContentTab.VideoFeed=Video feed
 SharedContentTab.VideoStream=Video stream
 SharedContentTab.VirtualFolders=Virtual folders
 SharedContentTab.WebContent=Web content
+Search.AllFiles=all files
+Search.ByAlbum=by album
```

Comment:
Is there a reason for these translations being in lowercase?

Reply:
Actually, this is just a visual aspect. For me, it did look better in the UI. Please feel free to change it!